src/View/Helper/FormHelper.php

```diff
@@ -150,6 +150,8 @@ class FormHelper extends Helper
             'confirmJs' => '{{confirm}}',
             // selected class
             'selectedClass' => 'selected',
+            // required class
+            'requiredClass' => 'required',
```

Comment:
Why does this need to be a template?

I guess I have to generally ask why classes exist as templates. I've never really noticed the `selectedClass` template before 🤔

Reply:
The use-case is users using CSS methodologies like BEM and wanting to make the HTML generated by CakePHP to conform to those conventions.

Reply:
Sure, I get why one would want to be able to change the class, I'm just wondering why this would need to be a template instead of an option like `errorClass`?

I'm already confused by `selectedClass` being a template, it looks workaround-ish to make it available to widgets, but that could've also been achieved by passing an option into the widgets `render()` method?